### ISSUE TITLE
Move abort no explination

### ISSUE THREAD

**Issue by Eldrox** [OLSF/libra#844]
# 🐛 Bug

## To reproduce
Validator node not syncing correctly, and then order is not correct. The explanation is not very useful, there is nothing to work from to correct it. surely it should have an explanation. 

**Stack trace/error message**
Message: Rejected with code: MoveAbort { location: "00000000000000000000000000000001::TowerState", abort_code: 130109, explanation: None }

**Comment by 0o-de-lally** [comment]
Yes the `tower` app should return an error for code 130109 which means there is a discontinuity in the proof being added to the tower

<img width="780" alt="Screen Shot 2021-12-03 at 2 31 08 PM" src="https://user-images.githubusercontent.com/1364012/144662017-79450050-c9f3-4543-8255-a191e4dc1a07.png">
